graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationSearchUtils.java

```diff
@@ -184,6 +187,11 @@ ImmutableList<EventWithContext> eventsFromAggregationResult(EventFactory eventFa
                     .flatMap(modifier -> modifier.eventModifierData(result.additionalResults()).entrySet().stream())
                     .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
 
+            // If an event processor specific event decorator is supplied, apply that decoration to the event
+            if (eventDecorator != null) {
```

Comment:
We can remove the `null` check when we make `eventDecorator` non-null as mentioned earlier.